src/bundle/DependencyInjection/Compiler/CronJobCompilerPass.php

```diff
@@ -39,10 +39,15 @@ public function process(ContainerBuilder $container)
                     ? $cronJob['category']
                     : CronJobsRegistry::DEFAULT_CATEGORY;
 
+                $cronJob['options'] = isset($cronJob['options'])
+                    ? ' '.$cronJob['options']
+                    : '';
+
                 $registry->addMethodCall('addCronJob', [
                     $reference,
                     $cronJob['schedule'],
                     $cronJob['category'],
+                    $cronJob['options'],
```

Comment:
If we add a space to command (different comment), we will not need the ternary above. All we will need is this:
```suggestion
                    $cronJob['options'] ?? '',
```

Reply:
@homarx Could you address it? Then I can pass it to QA.

Reply:
But wouldn't there be an unnecessary space in the command if no options are configured (options='')?